cypress/utils/utils.ts

```diff
@@ -46,3 +46,15 @@ export function base64DecodeToBuffer(string: string) {
 export function base64Decode(string: string) {
   return !string ? string : base64DecodeToBuffer(string.replace(/[-_]/g, char => char === '-' ? '+' : '/')).toString();
 }
+
+export const nodes = {
+  filterWitnessNode: (hosts: {name: string, witnessNode: boolean}[]) => {
```

Comment:
`filterWitnessNode()` and `witnessNode` config feild are not used in the code. 
Are they prepare for future test cases use?

Reply:
Yes, I would keep this code for future test cases.